terminologies/technique/averageLinkageHierarchicalClustering.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/technique/averageLinkageHierarchicalClustering",

```

Comment:
for consistency and how it is usually used this should be just "average linkage clustering" (which is a clustering technique of the technique class hierarchical agglomerative clustering)